docs/source/hyper-parameter-search.rst

```diff
@@ -236,8 +288,10 @@ train-and-score them until we find the best one.
 
     from dask_ml.model_selection import IncrementalSearchCV
 
-    search = IncrementalSearchCV(model, params, random_state=0)
-    search.fit(X, y, classes=[0, 1])
+    search = IncrementalSearchCV(model, params, 9, random_state=0)
+    _ = search.fit(X, y, classes=[0, 1])
```

Comment:
Style nit: I prefer a trailing a semicolon to hide the output if you don't want it shown (agreed that we don't want it shown here).

Reply:
Oooo that's nice. I didn't know that worked!